HoloStorageAccessor/internal/openapi/utils.go

```diff
@@ -0,0 +1,154 @@
+package openapi
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"log"
+	"net/url"
+	"os"
+	"path"
+	"strconv"
+	"strings"
+	"time"
+)
+
+type HologramQueryDetails struct {
+	IDs          []string
+	Mode         string
+	CreationMode string
+}
+
+type HologramPostInput struct {
+	Hologram Hologram
+	Author   Author
+	Patient  Patient
+}
+
+func ParseHologramUploadPostInput(formData url.Values) (HologramPostInput, error) {
+	var authorData Author
+	var patientData Patient
+	var hologramData Hologram
+
+	for key, _ := range formData {
```

Comment:
Would it make sense to actually use the values to, so that you don't have to `Get()` them individually when needed?

Reply:
I just remembered that formData.Get(key) returns "" when the values are not set. The values by themselves are in an array and would be harder to parse.